cmd/snap/notes.go

```diff
@@ -37,7 +60,48 @@ type Notes struct {
 	Broken   bool
 }
 
+func NotesFromRef(ref *snap.Ref) *Notes {
+	return &Notes{
+		DevMode: ref.Confinement != client.StrictConfinement,
```

Comment:
That looks a bit suspect. Shouldn't that be checking for actual devmode? This will catch classic as devmode too.